src/pkg/projects/netcoreapp/sfx/Microsoft.NETCore.App.SharedFx.sfxproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <SfxIdentity>Microsoft.NETCore.App</SfxIdentity>
+    <SfxPackageIdentity>$(SfxIdentity).Internal</SfxPackageIdentity>
+
+    <!-- Lay out the sfx in the legacy location for old packaging to keep working for now. -->
+    <SharedFrameworkLayoutDir>$(SharedFrameworkPublishDir)</SharedFrameworkLayoutDir>
+
+    <GenerateRuntimeGraph>true</GenerateRuntimeGraph>
+
+    <!-- These components are installed by the root shared framework, but not others. -->
+    <IncludeWerRelatedKeys>true</IncludeWerRelatedKeys>
+    <IncludeBreadcrumbStoreFolder>true</IncludeBreadcrumbStoreFolder>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PkgProjectReference Include="..\pkg\legacy\Microsoft.NETCore.App.Internal.pkgproj" />
+    <PkgProjectReference Include="..\pkg\legacy\Microsoft.NETCore.App.pkgproj" />
+    <PkgProjectReference Include="..\..\Microsoft.NETCore.DotNetHost\Microsoft.NETCore.DotNetHost.pkgproj" />
+  </ItemGroup>
+
+  <!--
```

Comment:
Are more changes coming here?

Reply:
Not as part of this migration. Added `This can be removed when the legacy infra is phased out.` to be more direct.